Orso.Arpa.Application/Localization/LocalizerCache.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Castle.Core.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Orso.Arpa.Domain.Entities;
+using Orso.Arpa.Domain.Interfaces;
+
+namespace Orso.Arpa.Application.Localization
+{
+    public class LocalizerCache
+    {
+        private static IServiceCollection _services;
+        private static readonly object _syncLock = new();
+        private static List<Translation> _translations;
+
+        public LocalizerCache(IServiceCollection services)
```

Comment:
Warum injzierst du hier nicht direkt den IArpaContext?

Reply:
Das hat einen etwas komplexeren Grund:
Es ist so, dass der LocalizerCache ein Singleton ist. Aus einem Singleton kann man keinen Scoped service injecten, welche ja der IArpaContext ist. Aus diesem Grund wird in dem seltenen Fall des Ladens der Translations ein Scope erstellt, aus welchem heraus das IArpaContext gezogen wird.